core/src/main/scala/cromwell/core/CromwellGraphNode.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.core
+
+import wdl4s.wdl.types.{WdlAnyType, WdlType}
+import wdl4s.wom.callable.Callable.OutputDefinition
+import wdl4s.wom.graph.GraphNode
+import wdl4s.wom.graph.GraphNodePort.{InputPort, OutputPort}
+
+// TODO WOM: https://github.com/broadinstitute/wdl4s/issues/193
+object CromwellGraphNode {
+  implicit class CromwellEnhancedGraphNode(val graphNode: GraphNode) extends AnyVal {
+    def unqualifiedName = graphNode.name
+    def fullyQualifiedName = graphNode.name
+    def relativeWdlType(node: GraphNode): WdlType = WdlAnyType
```

Comment:
Question: what's this `relativeWdlType` all about? In theory WOM should do all the type-relativising for us at GraphNode boundaries.

Reply:
yep this should go